app/locations/api.py

```diff
@@ -42,11 +34,6 @@ def suggest_new_POI(poi_suggestion: Dict[str, str], user: User, **kwargs):
     :param poi_suggestion: Suggestion to be saved
     :param user: User object of the user who submitted the suggestion
     """
-    try:
-        poi_suggestion["submitted_by"] = user.email
-        suggestion = new_POI_suggestion(poi_suggestion)
-        return None, 204
-    except InvalidPOISuggestionError as e:
-        return e.build_error()

```

Comment:
I think this is an interesting question of how much we should rely on the more "hidden" global catcher vs. catching them and managing them ourselves. Definitely a good discussion for in-person. For now this is great though.

Reply:
Thinking more on it I kinda like the idea that we can just code the happy path for the api methods and if the exception doesn't trigger a change in behavior we can just let is fall through to be caught by the global handler. I suppose it does introduce a bit of tribal knowledge to know it exists and _will_ be caught though.

Reply:
Yeah IMO for cases like this where all we're doing is building the error and returning it, there's no need to do it explicitly since the code is redundant at that point. But yeah for other cases where we wanna handle an error in a more specific way, we can still catch them in the endpoint function.